### DISCUSSION TITLE
Send different repositories' notifications to different emails set up for an account

### DISCUSSION THREAD

**Discussion by iddm** [community/community#39304] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi! I have many repositories and work within an organisation. If a repository is a part of an organisation, I'd like to receive email notifications to my working email, which is also connected to my account here, not to my private personal email. It might also be helpful to specify an email (connected to your account) for any repository if you want to subscribe to notifications. So, my proposal is every single time one presses the "Watch" button, allow not just to select the type of notifications but an email (one of the emails connected to the account currently being used) to which you want these notifications sent. A thing for the future is to figure out the possibilities of how to send or not send the push notifications to the phone application, as it is just one push id, regardless of how many emails you have.

Perhaps, the same thing can be done for watching separate issues, pull requests, discussions and so on, so, basically, any notifications.

**Comment by o-liver** [comment]
I have the same case: I want to send different email for work related repos, than for the repos I follow privately. I would also suggest that we can specify which email to use at the Watch button for a repository.
